src/api/src/lib.rs

```diff
@@ -236,6 +244,7 @@ mod tests {
                 vcpu_config: VcpuConfig { num: 1 },
                 block_config: None,
                 net_config: None,
+                console_config: None
```

Comment:
We should update the test to have a console_config that's being parsed as well. Best would be to have it with all possible values.